index.html

```diff
@@ -289,11 +289,57 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-    </main>
 
-    <script
-      src="https://kit.fontawesome.com/e966a3bd22.js"
-      crossorigin="anonymous"
-    ></script>
+
+    <footer id="Contact">
+      <section class="head">
+        <h2>
+          I'm always <br />
+          interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+      </section>
+
+      <form class="form" action="https://formspree.io/f/xdobnpad" method="post">
+        <input type="text" id="name" name="user-name" maxlength="30" placeholder="Full Name" required />
+        <input type="text" id="email" name="user-email" maxlength="500" placeholder="Email Address" required />
+        <textarea
+          name="message"
+          id="msg"
+          maxlength="500"
+          placeholder="Enter Text Here"
+          required
+        ></textarea>
+        <button id="form-button" type="submit">Get in Touch</button>
```

Comment:
- [ ] The button looks great :heart_eyes:  However, based on project requirements, the button's style should change on hover and active (when it's pressed). kindly implement the button interactions to match the Figma template :point_down: 
![Screenshot from 2022-05-20 11-12-08](https://user-images.githubusercontent.com/87965534/169498679-ee965fed-7430-450a-98d4-e4d9c90f22c3.png)
- [ ] Also, as you can see in the screenshot below :point_down: the color of your button doesn't match the one in the Figma template
![Screenshot from 2022-05-20 12-31-46](https://user-images.githubusercontent.com/87965534/169499563-c46f324b-bbc8-433e-91bf-a586762fdfc6.png)

:bulb:  You can press the button in the Figma and copy the exact same color in the inspect on the right :smile: 
![Screenshot from 2022-05-20 12-36-54](https://user-images.githubusercontent.com/87965534/169500442-32ddbdce-be64-40cc-b069-ae2e9de88dcc.png)